.buildkite/pipeline.yml

```diff
@@ -0,0 +1,41 @@
+common-params:
+  &publish-android-artifacts-docker-container
```

Comment:
💄  Nit: since this Docker image is not only used for the publish step, but also the lint and test steps, not sure why the anchor name was chosen to be _this_ specific 🤷 

May I suggest simply using `&docker-container` for this instead?

Reply:
I keep going back and forth on that decision, but the main reason this specific name is picked (besides being consistent with other projects) is that the AWS credentials are passed through to the Docker container. Although it's very unlikely that we'd accidentally leak them, I felt that by choosing a specific name and having the comment above the credentials may lead us (mostly myself) to be more cautious.

If that doesn't sound like a reasonable precaution, I don't mind renaming it to `docker-container`. As I said, I have been feeling unsure about this each time I replicated it in another project, so it'd be very easy to change my mind :)

Reply:
Ah I see, I didn't realize that was the intention behind the naming. Personally that name didn't really make me think "I'm announcing that S3 credentials are in this image" so that wouldn't have made the expected effect on me tbh 🙃 

If you want to highlight that aspect, maybe `docker-container-with-s3-tools` or `docker-container-with-s3-access` would make it clearer?

But again, it's really just a minutiae and probably not worth debating over much; I'd say just go with your guts of choose what makes sense for you, and don't let this block this PR from being merged 🙂

Reply:
Since this is the name we use in other projects, let's keep it as is then. We can re-visit them all together in the future. _And that's not because I am on a different computer and can't make changes right now 🤫_